_posts/blog/2022-09-09-our-covid-19-policy.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: blog
+categories: 
+  - blog
+title: "Our COVID-19 Policy"
+description: "At Chi Hack night we continue to believe in the importance of hosting in-person events particularly in a time where meaningful social life is so vital, but we must work together to do so safely. We recognize our civic responsibility of standing in solidarity with marginalized communities, particularly as they have been disproportionately impacted by the pandemic. As such, we have created a policy for protecting our community from COVID-19."
+date: 2022-09-09
+image: /images/logo/logo-star-social.jpg
+author: "Cameron Sow"
+author_url: https://www.cameronsow.com
+author_image: /images/people/board/cameron.jpg
+author_bio: "Bard Member, Chi Hack Night"
```

Comment:
typo "Board member"

Reply:
good catch! fixed in https://github.com/chihacknight/chihacknight.org/commit/f2cc30aff3c74560b0d8b570f449ece9ad0f8be2